packages/grafana-perf-tests/src/object-store-test.ts

```diff
@@ -0,0 +1,145 @@
+import { SharedArray } from 'k6/data';
+import execution from 'k6/execution';
+import grpc from 'k6/net/grpc';
+
+import { GRPCObjectStoreClient } from './object-store-client';
+import { Data, prepareData } from './prepare-data';
+
+const grpcToken = __ENV.GRPC_TOKEN;
+const grpcAddress = __ENV.GRPC_ADDRESS;
+
+if (typeof grpcToken !== 'string' || !grpcToken.length) {
+  throw new Error('GRPC_TOKEN env variable is missing');
+}
+
+if (typeof grpcAddress !== 'string' || !grpcAddress.length) {
+  throw new Error('GRPC_ADDRESS env variable is missing');
+}
+
+const client = new grpc.Client();
+const objectStoreClient = new GRPCObjectStoreClient(client, grpcAddress, grpcToken);
+
+const data: Data = new SharedArray('data', () => {
+  return [prepareData(JSON.parse(open('../scripts/tmp/filenames.json')), 50)];
+})[0];
+
+const scenarioDuration = '2m';
+
+export const options = {
+  setupTimeout: '5m',
+  teardownTimeout: '5m',
+  noConnectionReuse: true,
+  scenarios: {
+    writer: {
+      exec: 'writer',
+      executor: 'constant-arrival-rate',
+      rate: 1,
```

Comment:
note that all the values (rate, vus, timeUnit) here are very low, as its currently exercising the dummy server. we will increase them once the [SQL backend PR](https://github.com/grafana/grafana/pull/56348) is merged